prow/manifests/overlays/metal3/config.yaml

```diff
@@ -932,6 +932,21 @@ presubmits:
     - name: gomod
       branches:
         - main
+      skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
+      decorate: true
+      spec:
+        containers:
+          - args:
+              - ./hack/gomod.sh
+            command:
+              - sh
+            env:
+              - name: IS_CONTAINER
+                value: "TRUE"
+            image: docker.io/golang:1.20
+            imagePullPolicy: Always
```

Comment:
I think "Always" is a bit aggressive for imagePullPolicy, it think this should be pulled only if the image is not present

Reply:
Noted. But there might be another reason, why we set `imagePullPolicy` to ` Always` for all images. I can test it in separately.

Reply:
changed to IfNotPresent. It will prevent downloading images for each prow test. Nice catch.